torchao/experimental/kernels/mps/src/lowbit.h

```diff
@@ -74,10 +74,15 @@ using DispatchFn =
 
 inline void linear_lowbit_quant_weights_mps_impl(
     id<MTLBuffer> a_buf,
+    size_t a_offset,
```

Comment:
Wouldn't it be better to have like a tuple of buf + offset?
I.e. in function with 10+ arguments its easy to make a mistake about arg order

Reply:
updated to using a pair!